macos/FluentUI/Button/Button.swift

```diff
@@ -154,6 +154,39 @@ open class Button: NSButton {
 		}
 	}
 
+	/// When set to a non `nil` value ,the Secondary Image will always be placed on the trailing edge of the button in both LTR and
+	/// RTL language settings. It is designed to remain independent of  the `imagePosition` value of the NSButton which deals
+	/// with the main Button `image`.
+	public var secondaryImage: NSImage? {
+			didSet {
+				guard oldValue != secondaryImage else {
+					return
+				}
+
+				guard let cell = cell as? ButtonCell else {
+					return
+				}
+				
+				// Handle replacement/removal of a previously set image
+				if secondaryImageView.isDescendant(of: self) {
+					secondaryImageView.removeFromSuperview()
+				}
+
+				if let secondaryImage = secondaryImage {
+					secondaryImageView = NSImageView(image: secondaryImage)
+					secondaryImageView.translatesAutoresizingMaskIntoConstraints = false
+					addSubview(secondaryImageView)
+					NSLayoutConstraint.activate([
+						secondaryImageView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -cell.horizontalPadding),
+						secondaryImageView.centerYAnchor.constraint(equalTo: centerYAnchor)
+					])
+					updateContentTintColor()
```

Comment:
To avoid needing to update contentTintColor on the rest of the button while it's not changing, can update `updateContentTintColor` to call a new `updateSecondaryImageTintColor` which sets secondaryImageView's contentTintColor to the button's contentTintColor. And then directly call into `updateSecondaryImageTintColor` here instead